rcl/test/rcl/test_wait.cpp

```diff
@@ -27,6 +30,8 @@
 #endif
 
 
+#define TOLERANCE 100000  // clock error tolerance in nanoseconds
```

Comment:
I guess that there isn't a precise enough conversion for this in the `rcl` macros, i.e. there is no `RCL_US_TO_NS`.